### ISSUE TITLE
New issue with netapp-cloudmanager_cifs_server

### ISSUE THREAD

**Issue by rbenigno** [NetApp/terraform-provider-netapp-cloudmanager#139]
Just started experience an issue with netapp-cloudmanager_cifs_server resources.  The resource is created with no issues, but subsequent runs error out every time.  This issue just started for me.  The "cifs doesn't exist" message can be found in the provider code...

**Creation**

```
  # module.cvo1.netapp-cloudmanager_cifs_server.cl-cifs[0] will be created
  + resource "netapp-cloudmanager_cifs_server" "cl-cifs" {
      + client_id              = "<REDACT>"
      + id                     = (known after apply)
      + is_workgroup           = true
      + server_name            = "ntapcvo"
      + workgroup_name         = "WORKGROUP"
      + working_environment_id = "vsaworkingenvironment-<REDACT>"
    }
```

**Future plan / apply**

```
│ Error: cifs doesn't exist
│ 
│   with module.cvo1.netapp-cloudmanager_cifs_server.cl-cifs[0],
│   on ../modules/gcp_cvo/main.tf line 44, in resource "netapp-cloudmanager_cifs_server" "cl-cifs":
│   44: resource "netapp-cloudmanager_cifs_server" "cl-cifs" {
```

--

```
2022-10-26T16:56:36.957-0700 [DEBUG] provider.terraform-provider-netapp-cloudmanager_v22.9.1: 2022/10/26 16:56:36 Found working environment: {ntapcvo vsaworkingenvironment-<REDACT> GCP false VSA svm_ntapcvo}
2022-10-26T16:56:37.277-0700 [DEBUG] provider.terraform-provider-netapp-cloudmanager_v22.9.1: 2022/10/26 16:56:37 Working environment {ntapcvo vsaworkingenvironment-<REDACT> GCP false VSA }
2022-10-26T16:56:37.277-0700 [DEBUG] provider.terraform-provider-netapp-cloudmanager_v22.9.1: 2022/10/26 16:56:37 API root = /occm/api/gcp/vsa
2022-10-26T16:56:37.589-0700 [ERROR] vertex "module.cvo1.netapp-cloudmanager_cifs_server.cl-cifs[0]" error: cifs doesn't exist
2022-10-26T16:56:37.589-0700 [ERROR] vertex "module.cvo1.netapp-cloudmanager_cifs_server.cl-cifs (expand)" error: cifs doesn't exist
2022-10-26T16:56:37.590-0700 [INFO]  backend/local: plan operation completed
2022-10-26T16:56:37.591-0700 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2022-10-26T16:56:37.593-0700 [DEBUG] provider: plugin process exited: path=.terraform/providers/registry.terraform.io/netapp/netapp-cloudmanager/22.9.1/linux_amd64/terraform-provider-netapp-cloudmanager_v22.9.1 pid=75980
2022-10-26T16:56:37.593-0700 [DEBUG] provider: plugin exited
```

**Comment by chuyich** [comment]
DEVOPS-5656 for tracking.
Setup of CIFS configuration will be available only with CIFS server from Cloudmanager, creation of CIFS configuration with CIFS workgroup will be deprecated. Will fix on terraform resource accordingly.

**Comment by chuyich** [comment]
Fixed at 22.11.0
